extensions/amp-form/0.1/amp-form.js

```diff
@@ -918,15 +948,17 @@ export class AmpForm {
   /**
    * Transition the form to the submit success state.
    * @param {!Promise<!JsonObject>} jsonPromise
+   * @param {!ActionTrust} incomingTrust
    * @return {!Promise}
-   * @private visible for testing
```

Comment:
Why remove `visible for testing` ?

Reply:
The new hotness is `@visibleForTesting` which is meant to be used for public functions that aren't actually used externally. Not applicable here since it's a private function.